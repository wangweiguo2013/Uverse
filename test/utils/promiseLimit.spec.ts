import { assert, describe, expect, it } from 'vitest'

describe('promise limit ', () => {
  it('test', () => {
    assert.equal(3,3)
  })
  it('snapshot', () => {
    const text = 'whatever'
    expect(text).toMatchSnapshot()
  })
})