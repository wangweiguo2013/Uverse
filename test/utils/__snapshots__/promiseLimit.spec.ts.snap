// Vitest Snapshot v1

exports[`promise limit  > snapshot 1`] = `"whatever"`;
